source/content/debug-cache.md

```diff
@@ -48,53 +48,52 @@ Generated by WordPress or Drupal to send data back to the browser. The platform
 </dl>
 
 ## Debug Caching Issues
-
 Understand caching behavior for a given page by analyzing cache related HTTP headers from the command line with `curl`. For example:
 
-```bash{outputLines: 2-20}
+```bash{outputLines: 2-18}
 curl -I https://www.example.com
 HTTP/2 200
-cache-control: public, max-age=600 // highlight-line
+cache-control: public, max-age=0 // highlight-line
 content-type: text/html; charset=UTF-8
 server: nginx
-//highlight-start
-set-cookie: cookielawinfo-checkbox-necessary=yes; expires=Thu, 20-Feb-2020 17:31:51 GMT; Max-Age=3600; path=/
-set-cookie: cookielawinfo-checkbox-non-necessary=yes; expires=Thu, 20-Feb-2020 17:31:51 GMT; Max-Age=3600; path=/
-//highlight-end
 x-pantheon-styx-hostname: styx-fe1-a-789d66bff9-tztp6
 x-styx-req-id: 7f93c166-53fe-11ea-803e-b26d7703e33f
-date: Thu, 20 Feb 2020 16:31:51 GMT
-x-served-by: cache-mdw17379-MDW, cache-chi21146-CHI
-x-cache: MISS, MISS
-x-cache-hits: 0, 0
-x-timer: S1582216311.492451,VS0,VE204
-vary: Accept-Encoding, Cookie, Cookie
+date: Tue, 08 Dec 2020 16:41:37 GMT
+x-served-by: cache-mdw17348-MDW
+x-cache: MISS
+x-cache-hits: 0
+x-timer: S1607445693.448541,VS0,VE4134
+vary: Accept-Encoding, Cookie
+x-robots-tag: noindex
 age: 0 // highlight-line
 accept-ranges: bytes
 via: 1.1 varnish
+content-length: 41278
 ```
 
-If you see `Age: 0` after multiple requests, your site is not caching properly.
+This specific example is from a Drupal site that has anonymous caching enabled, but has "none" set for expiration of cached pages within Drupal's performance settings (`/admin/config/development/performance`). For details, see [Drupal Performance and Caching Settings](/drupal-cache).
```

Comment:
@alexfornuto I didn't link to a specific section because then I'd have to assume D7 vs D8. I mean to direct the reader to check this specific config: 

Drupal 8: https://pantheon.io/docs/drupal-cache#caching
Drupal 7: https://pantheon.io/docs/drupal-cache#minimum-cache-lifetime